hw/dv/tools/xcelium/common.ccf

```diff
@@ -29,13 +29,12 @@ set_branch_scoring
 // Scores statements within a block.
 set_statement_scoring
 
-
 // Enables Toggle scoring and reporting of SystemVerilog enumerations and multidimensional static
 // arrays , vectors, packed union, modport and generate blocks.
 set_toggle_scoring -sv_enum enable_mda -sv_struct_with_enum -sv_modport -sv_mda 16 -sv_mda_of_struct -sv_generate -sv_packed_union
 
 // Enables scoring of reset states and transitions for identified FSMs.
-set_fsm_reset_scoring
+// set_fsm_reset_scoring
```

Comment:
If I have this line in, it indicates I don't cover that every state goes back to reset if reset is pulled. The state machine is a primitive so I didn't think I should need to verify this.

Reply:
I don't think I agree with @weicaiyang comments or maybe I just don't understand. It looks like @msfschaffner replaced original fsm design code with this macro/primitive. clk_i/rst_ni aren't even explicitly referenced in the design, they go into the primitive. I was just trying to get the fsm coverage up. Is fsm coverage included in the V2 code coverage requirement of 90%?

Reply:
yes, we need to have 90% for FSM coverage. But if it's a reset related transition, it may be OK to waive them for V2. Just comment in the checklist, like [this](https://github.com/visbaz/opentitan/blob/e5d982e719bfb03944511e61572dca185ac48f9a/hw/ip/spi_host/doc/checklist.md?plain=1#L190
)

@msfschaffner's macro is to fix this #11607. 
Can we point me the primitive you mention? I don't think we can have a generic primitive for all kinds of FSM.
I still think the FSM is coded [here](https://github.com/lowRISC/opentitan/blob/6faf7ef55e791dc1c3c7beb3683ff7a95f0c1bef/hw/ip/csrng/rtl/csrng_main_sm.sv#L52), which has the actual design function

Reply:
Still not sure about the primitive: `PRIM_FLOP_SPARSE_FSM/prim_sparse_fsm_flop.You're right the fsm functionality is outside of it, it just defines the flops but it handles reset because there isn't a reset state defined. Anyway leaving that ccf option for now. I'm going to try to get the reset coverage up another way but I want the rest of this PR to get in now.

Reply:
sounds good.